src/util/util.go

```diff
@@ -77,6 +78,9 @@ var missingOpenBrowserCommandMessages = []string{
 // GetOpenBrowserCommand returns the command to run on the console
 // to open the default browser.
 func GetOpenBrowserCommand() string {
+	if runtime.GOOS == "windows" {
+		return "explorer"
+	}
```

Comment:
Why was this not added to the list of openBrowserCommands?

Reply:
Windows does not have a `which` command, but always an `explorer` command.